core/lib/mno_enterprise/concerns/models/organization.rb

```diff
@@ -70,6 +70,11 @@ def active?
     !self.account_frozen
   end
 
+  # Visible organizations for the dashboard - active orgs (not frozen) and orgs being frozen
+  def visible?(user)
+    self.active? || self.freezing && user.role_from_id(self.id) == "Super Admin"
```

Comment:
why only the Super Admin can see the Organizations ?

Reply:
Indeed only the "Super Admin" can see the freezing orgs. I will add parenthesis